tests/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "commerce-sdk-integration-test",
+  "version": "1.4.1-beta.0",
+  "description": "Integration test package for Salesforce Commerce SDK",
+  "private": true,
+  "scripts": {
+    "redis": "docker rm --force redis-commerce-sdk; docker run -dp 6379:6379 --name redis-commerce-sdk redis:5-alpine",
+    "redis:shutdown": "docker rm --force redis-commerce-sdk",
+    "redis:debug": "docker rm --force redis-commerce-sdk-debug; docker run -dp 6379:6379 --name redis-commerce-sdk-debug redis:5-alpine && docker exec -it redis-commerce-sdk-debug redis-cli monitor",
+    "redis:debug:shutdown": "docker rm --force redis-commerce-sdk-debug",
+    "test:basic": "mocha \"test/basic/*.test.js\" --retries 2 --timeout 5000",
+    "test:redis": "mocha \"test/redis/*.test.js\" --retries 2 --timeout 5000",
+    "pack:sdk": "npm pack ../packages/core && npm pack ../packages/generator && npm install",
+    "test": "npm run test:basic && npm run test:redis",
+    "postinstall": "depcheck"
+  },
+  "mocha": {
+    "forbidOnly": true,
+    "colors": true
+  },
+  "eslintConfig": {
+    "extends": [
+      "plugin:jsdoc/recommended",
+      "plugin:prettier/recommended"
+    ],
+    "plugins": [
+      "header",
+      "jsdoc"
+    ],
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "rules": {
+      "header/header": [
+        2,
+        "block",
+        [
+          "",
+          {
+            "pattern": "^ \\* Copyright \\(c\\) \\d{4}, salesforce.com, inc\\.$",
+            "template": " * Copyright (c) 2020, salesforce.com, inc."
+          },
+          " * All rights reserved.",
+          " * SPDX-License-Identifier: BSD-3-Clause",
+          " * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause",
+          " "
+        ]
+      ],
+      "jsdoc/require-description": [
+        "error",
+        {
+          "checkConstructors": false
+        }
+      ],
+      "jsdoc/require-jsdoc": [
+        "error",
+        {
+          "publicOnly": true,
+          "require": {
+            "ArrowFunctionExpression": true,
+            "ClassDeclaration": true,
+            "ClassExpression": true,
+            "FunctionDeclaration": true,
+            "FunctionExpression": true,
+            "MethodDefinition": true
+          }
+        }
+      ],
+      "jsdoc/require-param-name": "error",
+      "jsdoc/require-returns-check": "error",
+      "jsdoc/require-returns-description": "error"
+    }
+  },
+  "devDependencies": {
+    "@commerce-apps/core": "file:commerce-apps-core-1.4.1-beta.0.tgz",
```

Comment:
Dan has this as a dependency instead of a dev dependency.  Honestly not sure what makes more sense here, but it probably doesn't matter.